Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -41,6 +42,14 @@ open class Player: AVPlayerViewController {
         }
     }
 
+    override open var preferredFocusEnvironments: [UIFocusEnvironment] {
+        if let view = nextFocusEnvironment as? UIView, view.alpha > 0.1 {
```

Comment:
I consider that the `view.alpha > 0.1` check is not applicable to all `UIView`, it only works for an specific subset of `UICorePlugins`.

Reply:
We create a generic abstraction of the properties that make the focus engine can not focus a view, please check it again.